src/main/java/tools/MaplePacketCreator.java

```diff
@@ -1439,7 +1439,12 @@ private static void encodeParentlessMobSpawnEffect(MaplePacketLittleEndianWriter
         
         private static void encodeTemporary(MaplePacketLittleEndianWriter mplew, Map<MonsterStatus, MonsterStatusEffect> stati) {
                 int pCounter = -1, mCounter = -1;
-            
+
+                stati = stati.entrySet()  // to patch some status crashing players
+                        .stream()
+                        .filter(e -> !(e.getKey().equals(MonsterStatus.WATK) || e.getKey().equals(MonsterStatus.WDEF)))
+                        .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
```

Comment:
`Collectors` is not imported, so this does not compile. You should get into the habit of actually running and testing your code before submitting it in a PR.

I fixed the import and tested this myself, and it looks good. Previously the hazard damage displayed as "Miss", and now it shows the damage. Great fix, thank you.

Reply:
I admit I did not compile the code within Cosmic's proposed stacks, but the code have been tested and run proper before proposing this PR. Will take into consideration finer reviews at next code exchanges.
Glad it worked!